File: 프로그래머스/2/181188. 요격 시스템/요격 시스템.py
def solution(targets):
    targets.sort(key = lambda x:x[1])
    
    answer = 0
    point = 0
    for a, b in targets:
        if not a < point < b:
            point = b - 0.5
            answer += 1
        else:
            continue
    return answer

File: 프로그래머스/2/181188. 요격 시스템/요격 시스템.js
function solution(targets) {
    targets.sort((a, b) => (a[1] - b[1]));
    
    let last = targets[0][1];
    let answer = 1;
    
    for(let i = 1; i < targets.length; i++){
        if(last <= targets[i][0]){
            answer++;
            last = targets[i][1];
        }
    }
    return answer;
}